### PR TITLE
PATCH /api/articles/:article_id Ticket 8

### PR DESCRIPTION
**Pull request by xtej4ik**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -283,4 +283,81 @@ describe("POST /api/articles/:article_id/comments", () => {
...
+      .patch(`/api/articles/${articleId}`)
+      .send({ inc_votes: inc })
+      .expect(400)
+      .then((res) => {
+        const { body } = res;
+        expect(body.msg).toBe('Invalid vote value');
+    });
+  });
+
+});
```
Comment (Hard-Luck) [inline __tests__/app.test.js:374]: we should test for missing/incorrect body

**Review by Hard-Luck — COMMENTED** [review]
We just need one more test for if the response body is incorrect then you can merge

File: __tests__/app.test.js (hunk 1 above)
Comment (xtej4ik) [reply __tests__/app.test.js:374]: added! Thanks!

**Review by pippaamy — APPROVED** [review]
great work, lets merge it!

**Pull request merged**